skymarshal/skycmd/oidc_flags.go

```diff
@@ -79,6 +80,7 @@ func (flag *OIDCFlags) Serialize(redirectURI string) ([]byte, error) {
 		RedirectURI:               redirectURI,
 		InsecureEnableGroups:      !flag.DisableGroups,
 		InsecureSkipEmailVerified: flag.InsecureSkipEmailVerified,
+		GetUserInfo:               flag.GetUserInfo,
```

Comment:
Do you think it is really necessary to provide this configure or we can just set it to true? IMO, groups claim is always (for most users) something we want for Concourse auth for teams. 

Would `oidc.provider.UserInfo()` require certain config in IDP (e.g. Okta) to enable the userinfo endpoint? Would there be any risk by always quering this endpoint?

Reply:
I think we could always set it to `true`. I added it as a flag in case it causes any edge case issues so there's always the option of disabling it. I don't think you need to enable or configure anything in Okta to enable the userinfo endpoint.

Reply:
I'm happy to set it to `true` if you want

Reply:
hmm, with further looking into the OpenID spec and the impl of oidc.UserInfo(). I think your concern of the edge case is valid. Though [userinfo endpoint](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo) is a standard, there might still be issues when querying it, depends on the IDP server.

In this case, I think we could make the flag to
```suggestion
		GetUserInfo:               !flag.DisableGetUserInfo,
```